app/views/curation_concerns/base/_attributes.html.erb

```diff
@@ -9,5 +9,17 @@
     <%= presenter.attribute_to_html(:embargo_release_date, render_as: :date) %>
     <%= presenter.attribute_to_html(:lease_expiration_date, render_as: :date) %>
     <%= presenter.attribute_to_html(:rights, render_as: :rights) %>
+    <% if defined?(presenter.member_of_collection_ids) && presenter.member_of_collection_ids.size > 0 %>
```

Comment:
You are querying `presenter.member_of_collection_ids` but down below are using `presenter.member_of_collection_presenters`. I believe the guarding if statement should reflect the method used below.

Option 1:

```ruby
if presenter.try(:member_of_collection_presenters).present?
```

Option 2:

```ruby
defined?(presenter.member_of_collection_presenters?) && presenter.member_of_collection_presenters?
```

Other options?